pkg/client/unversioned/request.go

```diff
@@ -604,7 +636,17 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 	if client == nil {
 		client = http.DefaultClient
 	}
+
+	time.Sleep(time.Duration(backoff.calculateBackoff(url)) * time.Second)
 	resp, err := client.Do(req)
+	updateUrlMetrics(r, resp, err)
```

Comment:
@timothysc this hunk requires metrics patch, so its in this patch.  if i remove -> merge conflict gaurnateed.  assuming minmal rebasing required if i include...